src/test/scala/uk/gov/nationalarchives/dp/client/ContentClientTest.scala

```diff
@@ -49,108 +49,29 @@ abstract class ContentClientTest[F[_]](preservicaPort: Int, secretsManagerPort:
   val tokenResponse: String = """{"token": "abcde"}"""
   val tokenUrl = "/api/accesstoken/login"
 
-  "findExpiredClosedDocuments" should "search using the index names from the API" in {
-    val documentResponse = <DocumentsResponse xmlns="http://preservica.com/AdminAPI/v6.5">
-      <Documents>
-        <Document>
-          <Name>closure-result-index-definition</Name>
-          <ApiId>1</ApiId>
-        </Document>
-      </Documents>
-    </DocumentsResponse>
-    val documentContentResponse = <index>
-        <shortName>tns</shortName>
-        <term indexName="test_date_index" indexType="DATE"/>
-        <term indexName="test_string_index" indexType="STRING_DEFAULT"/>
-      </index>
+  "search" should "search using the parameters passed to the method" in {
     preservicaServer.stubFor(post(urlPathMatching(tokenUrl)).willReturn(ok(tokenResponse)))
-    preservicaServer.stubFor(get(urlPathMatching("/api/admin/documents")).willReturn(okXml(documentResponse.toString)))
-    preservicaServer.stubFor(
-      get(urlEqualTo("/api/admin/documents/1/content"))
-        .willReturn(okXml(documentContentResponse.toString))
-    )
     val searchResponse = """{"success":true,"value":{"objectIds":[],"totalHits":1}}"""
     val searchMapping: MappingBuilder = get(urlPathMatching("/api/content/search"))
       .willReturn(okJson(searchResponse))
     preservicaServer.stubFor(searchMapping)
 
-    valueFromF(client.findExpiredClosedDocuments())
+    valueFromF(client.search(SearchQuery("test", SearchField("test1", List("value1")) :: Nil)))
```

Comment:
Instead of `"test"` do you think it might be better to add a query that looks more like an actual one? Because it's hard to tell from Preservica's documentation what a query should look like.

Reply:
Yeah good point, I've put a minimal valid query string in the test.